src/main/java/com/faforever/client/i18n/I18nImpl.java

```diff
@@ -43,6 +43,11 @@ public Locale getUserSpecificLocale() {
     return this.userSpecificLocale;
   }
 
+  @Override
+  public String getCountryNameLocalized(String isoCode) {
+    return isoCode == null ? null : new Locale("", isoCode).getDisplayCountry(this.userSpecificLocale);
+  }
+
```

Comment:
I would prefer annotating with NonNull and checking for null before or by returning an optional instead. Then you could do ifPresent -> set Text to label. Setting a jfx label to null is not the way to go in my opinion. (An Optional could also easily be built using `Optional.ofNullable(isoCode).map(...)`)

Reply:
+1 for optional